server/tests/controller/user/profile.spec.js

```diff
@@ -92,7 +103,7 @@ describe("Test for User Profile", () => {
         .send(userData[2]);
       expect(res.statusCode).to.equal(200);
       expect(res.body.success).to.be.true;
-      expect(res.body.message).to.equal("User profile updated");
+      expect(res.body.message).to.eql("User profile updated");
```

Comment:
why did you change this please

Reply:
I thought that eql is more appropriate for testing strings